txjuju/cli.py

```diff
@@ -17,6 +17,31 @@
 from .errors import CLIError
 
 
+def _find_best_juju(supported):
+    for juju in supported:
+        try:
+            _utils.Executable.find(juju)
+        except _utils.ExecutableNotFoundError:
+            continue
+        else:
+            return juju
+    else:
+        return supported[0]
+
+
+JUJU1 = _find_best_juju([
```

Comment:
Can we get a test that modifies path and validates that JUJU1 and JUJU2 variables are properly discovered in environments which don't have juju-1 or juju-2 or juju-2.0